Source/CesiumRuntime/Public/CesiumGeoreference.h

```diff
@@ -707,17 +709,13 @@ class CESIUMRUNTIME_API ACesiumGeoreference : public AActor {
    */
   static FName DEFAULT_GEOREFERENCE_TAG;
 
-  GeoTransforms _geoTransforms;
+  mutable GeoTransforms _geoTransforms;
```

Comment:
Any chance we can get rid of this member completely?

Looks like it's created in `GetGeoTransforms`, but isn't updated anywhere else. We could just return a fresh copy every time that function is called.

Reply:
I think I originally avoided this because it's a breaking change to return GeoTransforms by value instead of const reference. But it's not a very serious one in the context of 2.0, so I did I as you suggested.

Reply:
Ok, I didn't realize that. But sounds like a minor breaking change, and another nudge to stop using this.